### ISSUE TITLE
Accessibility:[MAS4.2.12]:Screen reader is not announcing about the search results in Welcome page.

### ISSUE THREAD

**Issue by micahgodbolt** [microsoft/FluidFramework#4517]
### Repro Steps:

1. Navigate to the search field in the left nav and type something.
2. Now listen to the screen reader announcement.

### Actual Result
Screen reader is not announcing about the search results in Welcome page. It is simply staying mute after typing anything in the search field.

### Expected Result:
Screen reader should announce as “Search results available” and the number of search results.

### User Impact:

Users with visual impairment who are dependent on screen readers will not get to know whether search results are available or not.

**Comment by micahgodbolt** [comment]
I checked out the current storybook demo and it appears to be fixed. will look into updating storybook
